packages/cli/src/cmds/assert/assert.ts

```diff
@@ -0,0 +1,110 @@
+import process from 'process';
+import { verbose } from '../../utils';
+import ValidationError from '../error/validationError';
+import AbortError from '../error/abortError';
+import * as fs from 'fs';
+import 'fs/promises';
+import * as console from 'console';
+import { buildAppMap } from '../../search/utils';
+import AssertionChecker from './assertionChecker';
+import Assertion from './assertion';
+import { glob } from 'glob';
+import AssertionsConfig from './assertionsConfig';
+import ProgressFormatter from './formatter/progressFormatter';
+import PrettyFormatter from './formatter/prettyFormatter';
+
+exports.command = 'assert';
+exports.describe = 'Run assertions for AppMaps in the directory';
+
+exports.builder = (args) => {
+  args.option('appmap-dir', {
+    describe: 'directory to recursively inspect for AppMaps',
+    default: 'tmp/appmap',
+  });
+  args.option('config', {
+    describe: 'path to assertions config file',
+    default: './assertionsConfig',
+    alias: 'c',
+  });
+  args.option('format', {
+    describe: 'output format',
+    default: 'progress',
+    options: ['progress', 'pretty'],
+  });
+  return args.strict();
+};
+
+exports.handler = async (argv) => {
+  verbose(argv.verbose);
+
+  const commandFn = async () => {
+    const { appmapDir, config, format } = argv;
+
+    if (!fs.existsSync(appmapDir)) {
+      throw new ValidationError(
+        `AppMaps directory ${appmapDir} does not exist.`
+      );
+    }
+
+    let summary = { passed: 0, failed: 0, skipped: 0 };
+    const checker = new AssertionChecker();
+    const formatter =
+      format === 'progress' ? new ProgressFormatter() : new PrettyFormatter();
+    await import(config);
+    const assertions = new AssertionsConfig().assertions;
```

Comment:
I think this is only working because ./assertionsConfig is imported at the top of the file.